scripts/plugins/storage/sqlitelb.sh

```diff
@@ -244,20 +244,12 @@ sqlite_reset_db_fledge() {
         sqlite_log "info" "Building the metadata for the Fledge Plugin '${PLUGIN}' ..." "logonly" "pretty"
     fi
 
-    if [[ -f $DEFAULT_SQLITE_DB_FILE && $2 != "immediate" ]]; then
-    # Remove service schema files as per name
-    schema=$(${SQLITE_SQL} ${DEFAULT_SQLITE_DB_FILE} 'select name from service_schema;')
+    # 0- Remove service schema files
+    schema=`sqlite3 ${DEFAULT_SQLITE_DB_FILE} 'select name from service_schema;'`
     for f in $schema; do
-        echo "Removing $f service schema..."
-        echo "'${FLEDGE_DATA}/${f}.db'"
-        rm -f ${FLEDGE_DATA}/${f}.db*
-        echo "Removal of $f service schema Done!"
-        if [ -d "${FLEDGE_DATA}/buckets" ]; then
```

Comment:
Hi @ashwinscale , 

Why we are omitting removing buckets directory?  Aren't buckets supposed to be removed when we use **sqlitelb.sh** 
script.